### PR TITLE
Cleanup medium type related properties

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

File: core/common/src/main/java/alluxio/conf/PropertyKey.java [hunk 1]
```diff
@@ -2354,10 +2360,13 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.WORKER)
           .build();
-  public static final PropertyKey WORKER_TIERED_STORE_LEVEL1_DIRS_MEDIATYPE =
+  public static final PropertyKey WORKER_TIERED_STORE_LEVEL1_DIRS_MEDIUMTYPE =
       new Builder(Template.WORKER_TIERED_STORE_LEVEL_DIRS_MEDIUMTYPE, 1)
           .setDefaultValue("SSD")
```
Comment (calvinjia) [inline core/common/src/main/java/alluxio/conf/PropertyKey.java]: Should we derive this from the tier alias name as well?

File: core/common/src/main/java/alluxio/conf/PropertyKey.java [hunk 2]
```diff
@@ -2395,10 +2405,13 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.WORKER)
           .build();
-  public static final PropertyKey WORKER_TIERED_STORE_LEVEL2_DIRS_MEDIATYPE =
+  public static final PropertyKey WORKER_TIERED_STORE_LEVEL2_DIRS_MEDIUMTYPE =
```
Comment (calvinjia) [inline core/common/src/main/java/alluxio/conf/PropertyKey.java]: If there are more tiers, will the default still work?

File: core/common/src/main/java/alluxio/conf/PropertyKey.java (hunk 1 above)
Comment (apc999) [reply core/common/src/main/java/alluxio/conf/PropertyKey.java]: SG. updated

File: core/common/src/main/java/alluxio/conf/PropertyKey.java (hunk 2 above)
Comment (apc999) [reply core/common/src/main/java/alluxio/conf/PropertyKey.java]: no they will not work any more

File: core/common/src/main/java/alluxio/conf/PropertyKey.java [hunk 3]
```diff
@@ -2312,10 +2313,14 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.WORKER)
           .build();
-  public static final PropertyKey WORKER_TIERED_STORE_LEVEL0_DIRS_MEDIATYPE =
+  public static final PropertyKey WORKER_TIERED_STORE_LEVEL0_DIRS_MEDIUMTYPE =
       new Builder(Template.WORKER_TIERED_STORE_LEVEL_DIRS_MEDIUMTYPE, 0)
-          .setDefaultValue("MEM")
-          .setDescription("The media type for the top storage tier directories ")
+          .setDefaultValue(
+              String.format("${%s}", PropertyKey.WORKER_TIERED_STORE_LEVEL0_ALIAS.mName))
```
Comment (calvinjia) [inline core/common/src/main/java/alluxio/conf/PropertyKey.java]: In other places we use `Name.PROPERTY_KEY`, could we be consistent?

Comment (apc999) [reply core/common/src/main/java/alluxio/conf/PropertyKey.java]: fixed

**Comment by apc999** [comment]
@calvinjia  PTAL

**Comment by calvinjia** [comment]
alluxio-bot, merge this please.

**Comment by calvinjia** [comment]
alluxio-bot, cherry-pick this to branch-2.0 please.

**Pull request merged**